CONTRIBUTING_WITH_GIT2R.Rmd

```diff
@@ -0,0 +1,551 @@
+---
+title: "Contributing with git2r"
+# output: 
+#   md_document:
+#     variant: markdown_strict+backtick_code_blocks+autolink_bare_uris
+#     fig_width: 10
+#     fig_height: 6
+#     toc: true
+#     toc_depth: 2
+#   html_document:
+#     toc: true
+#     toc_depth: 2
+#     theme: spacelab 
+#     highlight: tango
+#     fig_width: 10
+#     fig_height: 6
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = TRUE)
+```
+
+# Introduction
+
+## Background/Motivation
+
+The **popgenInfo** project was designed to be an educational tool that can move
+at the same pace as method development by allowing people skilled in population
+genetics analysis to submit workflows that demonstrate the strengths of any
+given analysis in R. We use [git](https://git-scm.com/) and
+[GitHub](https://github.com) to accept contributions via pull requests using
+[GitHub flow](https://guides.github.com/introduction/flow/).
+
+> **What is git?**
+> 
+> Git is a *version control* program. It allows you to keep track of all changes
+that happen when you are writing, analyzing data, or coding. You can think of it
+as a sort of electronic lab notebook. Just like you would record any weights or
+measures carefully in your bench lab notebook, so should you record any analyses
+you do on the computer. With git, you "add" and "commit" changes that you make.
+These are analogous to writing down the values of your measurements and noting 
+how you took them. 
+
+The basic steps of the process are:
+
+1. **Fork** the [popgenInfo repository](https://github.com/NESCent/popgenInfo)\*
+2. **Clone** your fork to your local machine.\*
+6. Checkout your master branch, **fetch** the changes from NESCent and **merge**
```

Comment:
Intentional that 6. follows after 2.? I know that it will be rendered correctly anyway, but it sticks out when looking at the Rmd.